src/commands/branch.js

```diff
@@ -0,0 +1,31 @@
+const { flags } = require('@oclif/command');
+const AbstractAuthenticatedCommand = require('../abstract-authenticated-command');
+
+class BranchCommand extends AbstractAuthenticatedCommand {
+  async runIfAuthenticated() {
+    // eslint-disable-next-line no-unused-vars
+    const parsed = this.parse(BranchCommand);
+  }
+}
+
+BranchCommand.description = 'Create a new branch or list your existing branches';
+
+BranchCommand.flags = {
+  delete: flags.boolean({
+    char: 'd',
+    description: 'Delete the branch',
+  }),
+  project: flags.string({
+    char: 'p',
+    description: 'The name of the project to create a branch in',
+  }),
+  force: flags.boolean({
+    description: 'When deleting a branch, skip confirmation',
+  }),
```

Comment:
[MISC] I might be overly peaky about it but as the `--force` option is related to the delete one, I'd group them at the end:
- project
- delete
- force

Reply:
https://github.com/oclif/oclif/issues/222
`oclif` automatically order options by alphabetical order, grouped by `-` & `--` type of option :/